### PR TITLE
Create leiden-journal-of-international-law.csl

### PR DESCRIPTION
**Pull request by Danerad**
This is a work-in-progress citation style for Leiden Journal of International Law. 

I am mainly uploading this here because I am having issues syncing any changes with my team. I am relatively new to this so let me know if you'd like me to make changes.

### CODE REVIEW

File: leiden-journal-of-international-law.csl [hunk 1]
```diff
@@ -0,0 +1,637 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="note" version="1.0" default-locale="en-GB" and="symbol" et-al-min="3" initialize-with="." sort-separator=" " initialize-with-hyphen="false" demote-non-dropping-particle="never">
+  <info>
+    <title>Leiden Journal of International Law</title>
+    <title-short>LJIL</title-short>
+    <id>"http://www.zotero.org/styles/leiden-journal-of-international-law.csl</id>
```
Comment (rmzelle) [inline leiden-journal-of-international-law.csl]: There is a quote sign before the URL here that shouldn't be there.

File: leiden-journal-of-international-law.csl [hunk 2]
```diff
@@ -0,0 +1,637 @@
...
+    <link href="http://www.zotero.org/styles//leiden-journal-of-international-law.csl" rel="self"/>
```
Comment (rmzelle) [inline leiden-journal-of-international-law.csl]: `href="http://www.zotero.org/styles//leiden-journal-of-international-law.csl"` should be `href="http://www.zotero.org/styles/leiden-journal-of-international-law"`

File: leiden-journal-of-international-law.csl [hunk 3]
```diff
@@ -0,0 +1,637 @@
...
+    <link href="https://www.cambridge.org/core/services/aop-file-manager/file/575adb7dfcaa6b122ec738b3" rel="documentation"/>
+    <author>
+      <name>Øyvind Ege</name>
+      <email>oyvind.ege@gmail.com</email>
+    </author>
+    <category citation-format="note"/>
+    <category field="international law"/>
```
Comment (rmzelle) [inline leiden-journal-of-international-law.csl]: You can only pick from a defined set of categories: http://docs.citationstyles.org/en/stable/specification.html#appendix-i-categories. In this case, use "law".

File: leiden-journal-of-international-law.csl [hunk 4]
```diff
@@ -0,0 +1,637 @@
...
+    <issn>0922-1565</issn>
+    <eissn>1478-9698</eissn>
+    <summary>Citation as per LJIL standards</summary>
+    <published>May 2017</published>
```
Comment (rmzelle) [inline leiden-journal-of-international-law.csl]: This has to be a specially formatted timestamp, like the one you used for `<updated/>` (per http://books.xmlschemata.org/relaxng/ch19-77049.html). But we almost never use `<published/>` in this repository, so you can also just delete this line.

File: leiden-journal-of-international-law.csl [hunk 5]
```diff
@@ -0,0 +1,637 @@
...
+            <else>
+              <text value="Not supported"/>
+            </else>
+          </choose>
+        </else>
+      </choose>
+    </layout>
+  </citation>
+  <bibliography et-al-min="3" et-al-use-first="1" subsequent-author-substitute="&#8212;&#8212;">
+    <layout delimiter="; ">
```
Comment (rmzelle) [inline leiden-journal-of-international-law.csl]: You can't set a delimiter on `<layout/>` within `<bibliography/>`.

**Comment by rmzelle** [comment]
@Danerad, thanks for submitting this. Apart from my validation comments, could you also add a "template" link if you based this on an existing CSL style? See https://github.com/citation-style-language/styles/wiki/Style-Requirements#8---template-link.

Also, can you tell us which reference manager you're using this style with? Support for legal citation in CSL and most reference managers is rather limited, and there is a special variant of Zotero with improved legal support that uses a custom version of CSL: Juris-M (https://juris-m.github.io/). You might be interested in checking that out.

**Comment by Danerad** [comment]
Hi, thanks for checking this out. I have made the requested changes and validated the file properly - I wasn't sure how to use the validator without first uploading a file :)

I've largely created this from scratch using the CSL editor http://editor.citationstyles.org/visualEditor/, for use with **Mendeley**. I may migrate to Zotero in the future, especially if this project becomes unwieldy without Juris-M!

Thanks again, and let me know if you need anything else.

**Comment by rmzelle** [comment]
Thanks!

**Pull request merged**